guides/common/modules/proc_configuring-your-project-to-run-ansible-roles.adoc

```diff
@@ -18,6 +18,7 @@ endif::[]
 * `/usr/share/ansible/roles`
 * `/etc/ansible/collections`
 * `/usr/share/ansible/collections`
+* `(optional) VCS_INTEGRATION_PATH/ansible/roles`
```

Comment:
where does "VCS_INTEGRATION_PATH" come from? Is this something a user has to replace?